setup.py

```diff
@@ -119,7 +119,7 @@ def localversion_func(version: ScmVersion) -> str:
         "tqdm>=4.0.0",
         # torch 1.10 and 1.11 do not support quantized onnx export
         "torch>=1.7.0,!=1.10,!=1.11",
-        "transformers>4.0",
+        "transformers>=4.52.0",
```

Comment:
While we recommend people always be on latest transformers, we have decided to keep this pin loose
```suggestion
        "transformers>4.0",
```